packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -241,6 +246,15 @@ class SelectableRegion extends StatefulWidget {
   /// Called when the selected content changes.
   final ValueChanged<SelectedContent?>? onSelectionChanged;
 
+  /// {@template flutter.widgets.SelectableRegion.separator}
+  /// The separator to use when concatenating the selected text. Defaults to an
+  /// empty string, which means the selected text will be concatenated without
+  /// any separator. If the selectable text is arranged vertically, this
+  /// should be set to `\n` so that the selected text is copied with the
```

Comment:
Nit: I think this is better since it sounds less absolute.

"this should be set to" => "consider setting this to"